### ISSUE TITLE
Sample YAML files

### ISSUE THREAD

**Issue by vreuter** [epigen/looper#170]
Discussion with @nsheff led to possibility of generating (mostly, if not entirely) disjoint `Sample` YAML files, in which one is used for processing submission and the other stores information that's useful for working with the `Sample` post-processing, for further analyses. This would have a couple of benefits. 

It would bring the code into greater alignment with the model of usage and interaction that it's attempting to facilitate, and it would help with any lingering or future issues regarding the communication of `Project` information with a `Sample`. More significant on a longer-term basis is the platform that the post-processing file would provide for inclusion of tool-specific filepaths. If paths to outputs created by specific tools could be written to this secondary `Sample` file, it'd be easier for analyses in R or Python to use them with minimal additional effort from the analyst.

**Comment by vreuter** [comment]
I think that this isn't a major change and could be done as part of a final 0.7 version, prior to the full separation of `models` from `looper`. Sound good @nsheff ?

**Comment by nsheff** [comment]
My recollection of this issue is a bit different... I was thinking the post-processing `Sample` YAML would be a superset of the pre-processing one. Nothing disjoint here... so we may not be on the same page. Maybe I've forgotten what this was about.

**Comment by vreuter** [comment]
Maybe, but I think the differing conception may relate to https://github.com/epigen/looper/issues/150.
With that one in its current state, I'd be fine with post-processing one being superset of pre-processing one.

**Comment by nsheff** [comment]
Yeah, maybe right. Mostly I just want the post-processing one to be a complete representation of the sample.

but that's a pypiper issue I guess -- shouldn't pypiper be the one that outputs this? Or should it be looper?

**Comment by vreuter** [comment]
Yeah it should really be `pypiper` since pipelines can (and most do?) modify the `Sample`
